kuzzleio/kuzzle-backoffice
# Fix bugs

## What does this PR do ?

This PR fix the following issues : 
 - #350 (partially)
 - #354
 - #367
 - #394
 - #400
 - #402

I also remove old references to back-office.


### How should this be manually tested?

 - #350 : Create a collection with an `_` and try to access the dropdown menu
 - #354 : Create a collection with a mapping and read the hint about mapping to the right
 - #367 : Login, start to watch a collection, wait for token expiration (change `src/vuex/modules/auth/actions.js#L13` if you don't want to wait 2 hours) and the click on `Security` tab,the login modal should appear
 - #394 : Start watching a collection, download this [script](https://gist.github.com/Aschen/38e05560b46b1a121e45fc2f0feb3cb0/raw/8ba7343a72e33133a90e5c38d9b3b4a1363770aa/create-or-replace.js), run `node create-or-replace.js` twice, you should see the correct creation message
 - #400 : Go to `Users Management`, type `gordon` in the search bar, on an other tab create an user with id et username `gordon`, go to first tab and click `Search`, you should see you user
 - #402 : Create more than 10 profiles, create an user, you should see all profiles in the dropdown menu
